### ISSUE TITLE
could you please let me know what is the formula that calculates the number of flops related to the Batch normalization layer?

### ISSUE THREAD

**Issue by unipdjahanifar** [Mr-TalhaIlyas/Tensorflow-Keras-Model-Profiler#4]
Hi
could you please let me know what is the formula that calculates the number of flops related to the Batch normalization layer?
